website/docs/r/external_network_v2.html.markdown

```diff
@@ -0,0 +1,176 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_external_network_v2"
+sidebar_current: "docs-vcd-resource-external-network-v2"
+description: |-
+  Provides a VMware Cloud Director External Network resource (version 2). New version of this resource
+  uses new VCD API and is capable of creating NSX-T backed external networks as well as port group
+  backed ones.
+---
+
+# vcd\_external\_network\_v2
+
+Provides a VMware Cloud Director External Network resource (version 2). New version of this resource 
+uses new VCD API and is capable of creating NSX-T backed external networks as well as port group
+backed ones.
+
+~> **Note:** This resource uses new VMware Cloud Director
+[OpenAPI](https://code.vmware.com/docs/11982/getting-started-with-vmware-cloud-director-openapi) and
+requires at least VCD *10.0+*. (NSX-T *3.0+* requires VCD *10.1.1+*)
+
+Supported in provider *v3.0+*.
+
+## Example Usage (NSX-T backed external network)
+
+```hcl
+data "vcd_nsxt_manager" "main" {
+  name = "nsxManager"
+}
+
+data "vcd_nsxt_tier0_router" "router" {
+  name            = "first-router"
+  nsxt_manager_id = data.vcd_nsxt_manager.main.id
+}
+
+resource "vcd_external_network_v2" "ext-net-nsxt" {
+  name        = "nsxt-network"
```

Comment:
Is it `nsxt-network` or maybe more precise to name `nsxt-ext-net`?

Reply:
Improved names/comments.